002-db-schema.md

```diff
@@ -0,0 +1,88 @@
+# 002 - Template
+
+DB Schema by use cases from [001-usecases.md](./usecases.md)
+
+## Status
+
+* 2018-04-27: proposed
+* 2018-05-09: updated
+
+## Context
+
+By the defined use cases in the previous ADR to design a DB schema and describe all data
+
+## Decision
+
+* Use RDMS like Postgres
+* Initial SQL script like the following: 
+
+```sql
+CREATE TABLE person (
+  id SERIAL PRIMARY KEY,
+  login VARCHAR(20) NOT NULL,
```

Comment:
not sure about using `login`, `name` and `email` together

Reply:
why? what wrong with it?